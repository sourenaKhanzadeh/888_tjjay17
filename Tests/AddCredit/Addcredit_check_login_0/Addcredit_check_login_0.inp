Enter user type: AA
login 
admin
logout

Or 

Enter user type: FS,BS,SS
Login 
Standard user
Logout
