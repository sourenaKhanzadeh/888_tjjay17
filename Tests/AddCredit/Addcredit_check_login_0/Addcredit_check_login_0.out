Welcome to the add credit page
Enter session type: AA
Logged in as admin
Session terminated

or

Welcome to the add credit page
Enter session type: user standard(FS,BS,SS)
Logged in as user standard
Session terminated
